thrift/src/main/thrift/atoms/sidenote.thrift

```diff
@@ -0,0 +1,58 @@
+namespace * contentatom.sidenote
+namespace java com.gu.contentatom.thrift.atom.sidenote
+#@namespace scala com.gu.contentatom.thrift.atom.sidenote
+
+include "entity.thrift"
+include "../shared.thrift"
+
+enum SideNoteType {
+  QA = 0,
+  GUIDE = 1,
+  TIMELINE = 2,
+  PROFILE = 3
+}
+
+struct SideNoteAtom {
+  1: optional string typeLabel
```

Comment:
yes, it is the label above the title, like "Timeline" for instance. We expect eventually journalists will want to override the default value with something more specific.